cmd/influxd/upgrade/database.go

```diff
@@ -174,6 +187,16 @@ func upgradeDatabases(ctx context.Context, v1 *influxDBv1, v2 *influxDBv2, v1opt
 				log.Warn("Empty retention policy")
 			}
 		}
+
+		for _, cq := range db.ContinuousQueries {
+			if options.verbose {
+				log.Info("Exporting CQ", zap.String("db", db.Name), zap.String("cq_name", cq.Name))
+			}
+			_, err := cqFile.WriteString(fmt.Sprintf("%s\t%s\t%s\n", db.Name, cq.Name, cq.Query))
```

Comment:
Other ideas I had for the export format:
1. Mimic the pseudo-TSV output of `SHOW CONTINUOUS QUERIES`, which looks like:
    ```
    > SHOW CONTINUOUS QUERIES
    name: _internal
    name query
    ---- -----

    name: mydb
    name     query
    ----     -----
    other_cq CREATE CONTINUOUS QUERY other_cq ON mydb BEGIN SELECT mean(foo) INTO mydb.autogen.bar FROM mydb.autogen.baz GROUP BY time(1h) END

    name: telegraf
    name     query
    ----     -----
    cq_basic CREATE CONTINUOUS QUERY cq_basic ON telegraf BEGIN SELECT mean(usage_user) INTO telegraf.autogen.average_user_cpu FROM telegraf.autogen.cpu GROUP BY time(1m) END
    ```
2. Output InfluxQL-compatible lines, with one CQ per line (and any extra metadata in comments):
    ```
    -- some metadata we feel like adding
    CREATE CONTINUOUS QUERY cq_basic ON telegraf BEGIN SELECT mean(usage_user) INTO telegraf.autogen.average_user_cpu FROM telegraf.autogen.cpu GROUP BY time(1m) END
    -- some other metadata
    CREATE CONTINUOUS QUERY other_cq ON mydb BEGIN SELECT mean(foo) INTO mydb.autogen.bar FROM mydb.autogen.baz GROUP BY time(1h) END
    ```

Reply:
@timhallinflux do you have an preferences for the output format of the file? I'm inclined to write only the `CREATE CONTINUOUS` statements and nothing else.

Currently @danxmoran is writing a tab-delimited file as follows:

```tsv
database	name	query
telegraf	cq_basic	CREATE CONTINUOUS QUERY cq_basic ON telegraf BEGIN SELECT mean(usage_user) INTO telegraf.autogen.average_user_cpu FROM telegraf.autogen.cpu GROUP BY time(1m) END
```

Reply:
I would also prefer writing just the `CREATE CONTINUOUS` statements

Reply:
Tim voted in Slack for mirroring the `SHOW CONTINUOUS QUERIES` output, moving ahead with that option